src/main/java/com/code_intelligence/jazzer/mutation/support/TypeSupport.java

```diff
@@ -578,6 +579,9 @@ private static Annotation[] checkExtraAnnotations(
               .collect(Collectors.toCollection(HashSet::new));
       for (Annotation annotation : extraAnnotations) {
         boolean added = existingAnnotationTypes.add(annotation.annotationType());
+        if (annotation.annotationType().isAnnotationPresent(IgnoreRecursiveConflicts.class)) {
```

Comment:
Since the entire purpose of this function seems to be checking for the recursive conflict maybe we can exit early in the first lines of this function if `IgnoreRecursiveConflicts.class` is set?